index.html

```diff
@@ -424,5 +424,6 @@ <h4 class="desktop-header">Get started</h4>
       </div>
     </footer>
     <script src="menu.js"></script>
+    <script src="popup.js"></script>
```

Comment:
- [ ] The projects section should be created dynamically using JavaScript, kindly consider removing the project section from your HTML file from around line `100` to line `289`, and generate this section (Project section) dynamically on your script file